File: Snake with Arduino/BodySnake.cpp
/* Snake Class */
#include "BodySnake.h"
int _posX;
int _posY;

BodySnake::BodySnake() {

}

BodySnake::BodySnake(int posX, int posY) {
  _posX = posX;
  _posY = posY;
}

BodySnake::~BodySnake() {

}

int BodySnake::GetPosX() {
  return _posX;
}

int BodySnake::GetPosY() {
  return _posY;
}

void BodySnake::SetPosX(int posX) {
  _posX = posX;
}

void BodySnake::SetPosY(int posY) {
  _posY = posY;
}

File: Snake with Johnny Five/models/bodySnake.js
class BodySnake {
    constructor(posX, posY) {
        this._posX = posX;
        this._posY = posY;
    }

    GetPosX() {
        return this._posX;
    }

    GetPosY() {
        return this._posY;
    }

    SetPosX(posX) {
        this._posX = posX;
    }

    SetPosY(posY) {
        this._posY = posY;
    }
}

module.exports = BodySnake;